node/node_test.go

```diff
@@ -473,7 +476,6 @@ func TestNodeNewNodeCustomReactors(t *testing.T) {
 func TestNodeNewNodeDeleteGenesisFileFromDB(t *testing.T) {
 	config := test.ResetTestRoot("node_new_node_delete_genesis_from_db")
 	defer os.RemoveAll(config.RootDir)
-	fmt.Println(config.RootDir)
```

Comment:
This was accidentally left over from my last PR so I am piggy backing it here instead of opening a new PR.

Reply:
resolved